src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemUpdater.java

```diff
@@ -130,4 +127,16 @@ public void onVoltageControlModeChange(DiscreteVoltageControl voltageControl, Di
             }
         }
     }
+
+    @Override
+    public void onPhaseControlTapChange(DiscretePhaseControl phaseControl) {
```

Comment:
This should be handled in the `TargetVector` to just update the alpha1 target value. There is no need to change the equation system. => performance will also be better by just updating the target (no need to rebuild the jacobian matrix)

Reply:
I have made a fix and I think it is correct. I knew that it was not the good solution when updating the equation system. So I have used the listeners you have implemented and I am really impress by what you have implemented in the open-loadflow. It is really clever !